### ISSUE TITLE
Totally blank screen on opening ANY plist file

### ISSUE THREAD

**Issue by and3r00** [corpnewt/ProperTree#180]
I haven't been able to use this application to open a plist file successfully.
I am running OSX on a native ARM MacBook M3.
I'm not sure, but if I had to guess it is a font issue. 
I've included the same file within Xcode, showing it's not an empty file.
<img width="886" alt="Screenshot 2024-08-25 at 5 15 28 PM" src="https://github.com/user-attachments/assets/3481dbac-02d2-4311-a70d-8b12c951fdfd">

<img width="649" alt="Screenshot 2024-08-25 at 5 19 23 PM" src="https://github.com/user-attachments/assets/bb6b1d02-b14c-4f93-bd9d-2a5311125a4c">

**Comment by ai19880612** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by corpnewt** [comment]
What python version are you using?  The usual fix for nothing showing is to install the latest python version from python.org (you can find python 3.12.5 [here](https://www.python.org/downloads/release/python-3125/)).

-CorpNewt

**Comment by Calebh101** [comment]
Yeah this happened to me when running the included 3.9, fixed when upgrading to 3.13
Also yes I'm late
